Makefile

```diff
@@ -447,6 +450,11 @@ test-failing-prove: $(prove_failing_tests:=.prove)
 
 test-klab-prove: $(smoke_tests_prove:=.klab-prove)
 
+test-prove-optimizations: $(prove_optimization_tests:=.prove)
+
+tests/specs/evm-optimizations-spec.md: optimizations.md
```

Comment:
can you add a comment explaining how this file is generated?

Reply:
```suggestion
# to generate optimizations.md, run: ./optimizer/optimize.sh &> output
tests/specs/evm-optimizations-spec.md: optimizations.md
```